src/player.cpp

```diff
@@ -4130,7 +4130,7 @@ int player::overmap_sight_range(int light_level) const
         return 20;
     }
     else if (!(has_amount("binoculars", 1) || has_amount("rifle_scope", 1) ||
-        has_amount("survivor_scope", 1) || -1 != weapon.has_gunmod("rifle_scope") ) &&
+        has_amount("survivor_scope", 1) || has_bionic("bio_optic") || -1 != weapon.has_gunmod("rifle_scope") ) &&
```

Comment:
Ya I know it does. It might cause a conflict with my other CBM PR to. I forgot to write that on this PR last night. I can always change it later.